vignettes/razzo_pipeline.Rmd

```diff
@@ -30,21 +30,20 @@ Update our dependencies:
 ```{r message=FALSE, install_dependencies}
 if (!require(ape)) {install.packages("ape")}
 if (!require(mbd)) {devtools::install_github("Giappo/mbd", quiet = TRUE)}
-devtools::install_github("ropensci/beautier", quiet = TRUE)
-devtools::install_github("ropensci/tracerer", quiet = TRUE)
-devtools::install_github("ropensci/beastier", quiet = TRUE)
-devtools::install_github("ropensci/mauricer", quiet = TRUE)
-devtools::install_github("ropensci/babette", quiet = TRUE)
-devtools::install_github("richelbilderbeek/mcbette", quiet = TRUE)
-devtools::install_github("richelbilderbeek/pirouette", quiet = TRUE)
+# devtools::install_github("ropensci/beautier", quiet = TRUE)
+# devtools::install_github("ropensci/tracerer", quiet = TRUE)
+# devtools::install_github("ropensci/beastier", quiet = TRUE)
+# devtools::install_github("ropensci/mauricer", quiet = TRUE)
+# devtools::install_github("ropensci/babette", quiet = TRUE)
+# devtools::install_github("richelbilderbeek/mcbette", quiet = TRUE)
+# devtools::install_github("richelbilderbeek/pirouette", quiet = TRUE)
```

Comment:
I would be in favor of removing these lines altogether :rainbow:

Reply:
Agree :+1: